.github/workflows/build.yml

```
@@ -0,0 +1,41 @@
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}  # For PR decoration
+          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}    # SonarCloud auth
+        run: |
+          dotnet sonarscanner begin \
+            /k:"Learnathon-By-Geeky-Solutions_debugger-squad" \
+            /o:"Learnathon" \
+            /d:sonar.login="$SONAR_TOKEN" \
+            /d:sonar.host.url="https://sonarcloud.io" \
+            /d:sonar.exclusions="Ecommerce/Ecommerce.DataAccess/migrations/*" \
+            /d:sonar.scanner.scanAll=false
+          dotnet build Ecommerce/Ecommerce.sln
+          dotnet sonarscanner end /d:sonar.login="$SONAR_TOKEN"
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)